devices/ikea.js

```diff
@@ -775,6 +787,23 @@ module.exports = [
             await reporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['SYMFONISK sound remote gen2'],
+        model: 'E2123',
+        vendor: 'IKEA',
+        description: 'SYMFONISK sound remote gen2',
+        fromZigbee: [fz.battery, fz.command_toggle, fz.command_move, fz.command_step, ikea.fz.command_dots],
+        toZigbee: [tz.battery_percentage_remaining],
+        exposes: [
+            e.battery().withAccess(ea.STATE_GET),
```

Comment:
Please add `e.action(['dot_single', 'off', 'dot_double', 'dot_long'])` here.

Reply:
I'll add this line as soon as we resolve the other comment (L252+).

Reply:
Hi,
Also missing the actions for the other buttons:
- toggle
- brightness_move_up
- brightness_move_down
- brightness_step_up
- brightness_step_down